.code-samples.meilisearch.yaml

```
@@ -225,11 +223,11 @@ search_parameter_guide_filter_1: |-
     filter: 'title = Nightshift'
   })
 search_parameter_guide_filter_2: |-
-  client.index.search('n', {
+  client.index.search('shifu', {
```

```suggestion
  client.index('movies').search('shifu', {
```
At the same time, do we take the opportunity to put the name of the index?

---

Completely!! Thanks!